### ISSUE TITLE
HTTPS Certificat has expired/is invalid

### ISSUE THREAD

**Issue by JLLeitschuh** [json-schema-org/understanding-json-schema#72]
Open this link in chrome and you'll get a "this connection is not secure" warning:
https://json-schema.org/understanding-json-schema/

**Comment by Relequestual** [comment]
Thanks @JLLeitschuh.

We are aware of this and are trying to resolve it.
It's more complicated to resolve than you might appreciate, and I'd rather not go into details here.

**Comment by JLLeitschuh** [comment]
Fair enough, I just didn't see an issue for it, figured I'd let you know.

Good luck!

**Comment by mdboom** [comment]
I think this is now fixed... @Relequestual ?

**Comment by Relequestual** [comment]
Yeah, eventually got this resolved! =D
